octobot_commons/pretty_printer.py

```diff
@@ -150,8 +153,20 @@ def global_portfolio_pretty_print(
             available = f"({get_min_string_from_number(asset_dict[constants.PORTFOLIO_AVAILABLE])})"
             if markdown:
                 available = "{:<12}".format(available)
-
-            holding_str = f"{total} {available} {currency}"
+            ref_value = ""
+            if currency_values and ref_market_name:
+                try:
+                    ref_value = (
+                        f" ({get_min_string_from_number(currency_values[currency]*asset_dict[constants.PORTFOLIO_TOTAL])} {ref_market_name})"
+                        if ref_market_name != currency else ""
+                    )
+                    if markdown:
+                        ref_value = "{:<12}".format(ref_value)
+                except KeyError:
+                    # no currency value
+                    pass
+            
+            holding_str = f"{total} {available} {currency}{ref_value}"
```

Comment:
I'm wondering if printing the available amount is really necessary, maybe we should use the value in ref market instead and display the total locked funds in another command. What is you opinion on this ? 
I think we should not display all 3 at the same time because it makes it on multiple lines on telegram mobile which makes it harder to read

Reply:
ah yeah that makes sense, I'll swap it and we add the command later